src/main/java/io/vertx/starter/service/GeneratorService.java

```diff
@@ -174,6 +174,10 @@ private void createProject(VertxProject project, TempDir tempDir) throws IOExcep
   }
 
   private String packageName(VertxProject project) {
+    String packageName = project.getPackageName();
+    if (packageName != null && !packageName.trim().isEmpty()) {
```

Comment:
`StarterResource` defines a `isNotBlank` method. I'm sure is not the last time we'll have to do such snippet. Does it worth an `StringUtil` class?

Reply:
Not unbearable yet to me :wink: